server/controllers/routes/client/getTransactionById.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable camelcase */
+const transactions = require('../../../database/models/transactions');
+const exchangeMoney = require('../../../database/models/exchangeMoney');
+const client = require('../../../database/models/client');
+
+const getTransactionById = async (req, res) => {
+  try {
+    let promise = Promise.resolve();
+    const transactionTableData = await transactions.find({
+      client_id: res.clientId,
+    });
+    const result = [];
+    const exchangeDetail = [];
+    const clientData = await client.findById(res.clientId);
+    if (clientData.length === 0) throw new Error();
```

Comment:
try to handle this by a better way than throwing an error, for example, something like this 
```suggestion
    if (clientData.length === 0){ res.status(400).json({' The client is not exist`  });}
```